plugins/woocommerce/src/Admin/RemoteInboxNotifications/SpecRunner.php

```diff
@@ -119,7 +104,7 @@ private static function get_url( $action ) {
 	 * @returns object The locale that was found, or null if no matching locale was found.
 	 */
 	public static function get_locale( $locales ) {
-		$wp_locale           = get_locale();
+		$wp_locale           = get_user_locale();
```

Comment:
Change to [`get_user_locale`](https://developer.wordpress.org/reference/functions/get_user_locale/) so we can show different locale strings for different users.

Fallback to `get_locale()` when the user is not found.